extensions/wikia/Recirculation/js/views/mixedFooter.js

```diff
@@ -0,0 +1,132 @@
+define('ext.wikia.recirculation.views.mixedFooter', [
+	'jquery',
+	'wikia.window',
+	'wikia.tracker',
+	'ext.wikia.recirculation.utils',
+	'ext.wikia.recirculation.plista'
+], function ($, w, tracker, utils, plista) {
+	'use strict';
+
+	var track = tracker.buildTrackingFunction({
+			category: 'mixed-content-footer',
+			trackingMethod: 'analytics'
+		}),
+		$mixedContentFooter = $('#mixed-content-footer');
+
+	function render(data) {
+		var newsAndStoriesList = data.nsItems.items,
+			wikiArticlesList = data.wikiItems.items,
+			templateList = getTemplateList(newsAndStoriesList),
+			templates = {},
+			$discussions = $(data.discussions);
+		$('.mcf-discussions-placeholder').replaceWith($discussions);
+		$discussions.find('.discussion-timestamp').timeago();
+
+		return utils.loadTemplates(templateList)
+			.then(function (data) {
+				templateList.forEach(function (templateName, index) {
+					templates[templateName] = data[index];
+				});
+			})
+			.then(plista.prepareData(wikiArticlesList))
+			.then(function () {
+				injectTemplates(templates, newsAndStoriesList, wikiArticlesList);
+				setupTracking();
+			})
+	}
+
+	function injectTemplates(templates, newsAndStoriesList, wikiArticlesList) {
+		var $newsAndStoriesHook = $('.mcf-card-ns-placeholder'),
+			$wikiArticleHook = $('.mcf-card-wiki-placeholder');
+
+		$.each($newsAndStoriesHook, function (index) {
+			var $this = $(this),
+				template = templates['client/Recirculation_article.mustache'],
+				newsAndStoriesItem = newsAndStoriesList[index],
+				type = 'ns-article';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "type" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

Reply:
sonar says it is unused, I would remove it